person.rb

```diff
@@ -0,0 +1,29 @@
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(name: 'Unknown', age: nil, parent_permission: true)
+    @id = generate_id
```

Comment:
**OPTIONAL** - You have done well so far, I suggest that you generate a random id when creating a `person` class. Please fix this 🙏.
`@id = Random.rand(1..1000)`